hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/io/TestEnumSetWritable.java

```diff
@@ -118,7 +118,8 @@ public void testSerializeAndDeserializeNull() throws IOException {
   @Test
   public void testAvroReflect() throws Exception {
     String schema = "{\"type\":\"array\",\"items\":{\"type\":\"enum\","
-        + "\"name\":\"TestEnumSet\","
+        +   "\"name\":\"TestEnumSet\","
+        + "\"namespace\":\"org.apache.hadoop.io.TestEnumSetWritable\","
```

Comment:
Why is this line `+ "\"namespace\":\"org.apache.hadoop.io.TestEnumSetWritable\","` appearing twice?